src/pages/editItem.test.js

```diff
@@ -555,4 +562,216 @@ describe('<EditItem/>', () => {
+
+        expect(codeElementIdSchemeRadio).toBeChecked()
+
+        const attributeOrgUnitIdSchemeRadio = within(
+            screen.getByTestId('org-unit-id-scheme-selector')
+        ).getByRole('radio', { name: 'Attribute' })
+
+        expect(attributeOrgUnitIdSchemeRadio).toBeChecked()
+        expect(
+            within(screen.getByTestId('org-unit-id-scheme-selector')).getByText(
+                'Taco Friday'
+            )
+        ).toBeInTheDocument()
+
+        const idCategoryOptionComboSchemeRadio = within(
+            screen.getByTestId('category-option-combo-scheme-selector')
+        ).getByRole('radio', { name: 'ID' })
+
+        expect(idCategoryOptionComboSchemeRadio).toBeChecked()
+    })
+
+    it('does not post anything for ID schemes when None option is selected', async () => {
+        const request = testRequest()
+        const dataExchange = testDataExchange({
+            requests: [request],
+            targetType: 'INTERNAL',
+            inputIdSchemes: {
+                idScheme: 'UID',
+                dataElementIdScheme: 'CODE',
+                orgUnitIdScheme: 'UID',
+                categoryOptionComboIdScheme: 'CODE',
+            },
+        })
+
+        const { screen } = setUp(<EditItem />, {
+            userContext: testUserContext({ canAddExchange: true }),
+            dataExchange,
+        })
+
+        const user = userEvent.setup()
+
+        await screen.findByTestId('add-exchange-title')
+
+        const defaultElementIdSchemeRadio = within(
+            screen.getByTestId('element-id-scheme-selector')
+        ).getByRole('radio', { name: 'None (follows Input general ID scheme)' })
+        await user.click(defaultElementIdSchemeRadio)
+
+        const defaultOrgUnitIdSchemeRadio = within(
+            screen.getByTestId('org-unit-id-scheme-selector')
+        ).getByRole('radio', { name: 'None (follows Input general ID scheme)' })
+        await user.click(defaultOrgUnitIdSchemeRadio)
+
+        const defaultCategoryOptionComboSchemeRadio = within(
+            screen.getByTestId('category-option-combo-scheme-selector')
+        ).getByRole('radio', { name: 'None (follows Input general ID scheme)' })
+        await user.click(defaultCategoryOptionComboSchemeRadio)
+
+        const saveExchange = within(
+            screen.getByTestId('edit-item-footer')
+        ).getByText('Save exchange')
+        await user.click(saveExchange)
+
+        const expectedPayload = [
+            {
+                op: 'add',
+                path: '/target',
+                value: {
+                    type: 'INTERNAL',
+                    request: {
+                        idScheme: 'UID',
+                    },
+                },
+            },
+        ]
+
+        expect(patchExchange).toHaveBeenCalled()
+        expect(patchExchange).toHaveBeenCalledWith(expectedPayload)
+    }, 7000),
+        it('posts expected id schemes based on selections', async () => {
```

Comment:
I wanted this test to have a change to select an attribute, but I could not get the select to work as expected (e.g. when I click based on the label text, it does not appear to actually select the option in jest)